solution/0001.Two Sum/Solution.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+public class Solution {
+    public int[] TwoSum(int[] nums, int target) {
+        var dict = new Dictionary<int, int>();
+        for (var i = 0; i < nums.Length; ++i)
+        {
+            int index;
+            if (dict.TryGetValue(target - nums[i], out index))
+            {
+                return new [] { index + 1, i + 1};
```

Comment:
Expected [0, 1], but got [1, 2]:

![image](https://user-images.githubusercontent.com/21008209/73131290-6d5cb480-4043-11ea-83e1-fc92cd970faf.png)


```suggestion
                return new [] { index, i};
```

Reply:
Maybe my solution is too old and the test cases have updated. Let me run all my solutions on LeetCode again. Will update the PR later.

Reply:
Thank you :+1:

Reply:
Updated.